src/frontend/src/pages/FinancePage/ReimbursementRequestDetailPage/ReimbursementRequestDetailsView.tsx

```diff
@@ -69,6 +77,16 @@ const ReimbursementRequestDetailsView: React.FC<ReimbursementRequestDetailsViewP
     }
   };
 
+  const handleDeny = () => {
+    try {
+      denyReimbursementRequest();
```

Comment:
Make sure to test the case when the user is allowed to deny the certain RR. Currently, when the user tries to click 'YES' on RR deny modal, the modal does not close. I think this is b/c you forgot to include `setShowDenyModal(false)` in here `handleDeny` func. Also test if the RR has been actually marked denied after we click 'YES' in RR deny modal.